dev/com.ibm.ws.jbatch.wola/resources/com/ibm/ws/jbatch/wola/internal/resources/BatchWolaMessages.nlsprops

```diff
@@ -0,0 +1,41 @@
+###############################################################################
+# Copyright (c) 2014 IBM Corporation and others.
+# All rights reserved. This program and the accompanying materials
+# are made available under the terms of the Eclipse Public License v1.0
+# which accompanies this distribution, and is available at
+# http://www.eclipse.org/legal/epl-v10.html
+#
+# Contributors:
+#     IBM Corporation - initial API and implementation
+###############################################################################
+# # {0} description of each insert field
+# MSG_DESCRIPTIVE_NAME_CWSJX0000=CWSJX0000I: This is a message with inserts {0}
+# MSG_DESCRIPTIVE_NAME_CWSJX0000.explanation=Explanation text for the message
+# MSG_DESCRIPTIVE_NAME_CWSJX0000.useraction=User action text for the message
+#
+#CMVCPATHNAME com.ibm.ws.jbatch.wola/resources/com/ibm/ws/jbatch/wola/internal/resources/BatchWolaMessages.nlsprops
+#COMPONENTPREFIX CWWKY
+#COMPONENTNAMEFOR CWWKY WebSphere Batch
+#ISMESSAGEFILE TRUE
+#NLS_ENCODING=UNICODE
+#
+# NLS_MESSAGEFORMAT_VAR
+#
+#   Strings in this file which contain replacement variables are processed by the MessageFormat 
+#   class (single quote must be coded as 2 consecutive single quotes ''). Strings in this file 
+#   which do NOT contain replacement variables are NOT processed by the MessageFormat class 
+#   (single quote must be coded as one single quote '). 
+# -------------------------------------------------------------------------------------------------
+# 0250 - 0299: com.ibm.ws.jbatch.wola
+# 
+#
+# Note: do not translate "WOLA", "z/OS Local Adapter", or "batchManagerZos".
+batch.wola.listener.activation=CWWKY0250I: The batch listener for z/OS Local Adapter (WOLA) requests has been activated and is registered with JNDI name {0}
+batch.wola.listener.activation.explanation=The listener is active and ready to receive requests from clients using z/OS Local Adapters (WOLA), including the native z/OS batch utility batchManagerZos. 
+batch.wola.listener.activation.useraction=No action is required. 
+ 
+# Note: do not translate "WOLA", "z/OS Local Adapter", or "batchManagerZos".
+batch.wola.listener.deactivation=CWWKY0251I: The batch listener for z/OS Local Adapter (WOLA) requests has been deactivated.
+batch.wola.listener.deactivation.explanation=The listener has been deactivated and will no longer serve requests from clients using z/OS Local Adapters (WOLA), including the native z/OS batch utility batchManagerZos.
```

Comment:
Suggested changes similar to those suggested for the previous explanation

The listener is deactivated and no longer serves requests from clients that use WOLA, including the batchManagerZos native z/OS batch utility.

Or, if WOLA is not defined in the message:

The listener is deactivated and no longer serves requests from clients that use z/OS Local Adapters (WOLA), including the batchManagerZos native z/OS batch utility.